README.md

```diff
@@ -23,6 +23,19 @@ work in progress.
 Please file reports on the
 [GitHub Issue Tracker](https://github.com/dart-lang/dart-pad/issues).
 
+## Running locally
```

Comment:
This isn't necessary to run the front-end; you can launch web/index.html (the desktop version) or web/mobile.html (the mobile version) directly from the editor.

I think these instructions may confuse people a bit. Perhaps add a disclaimer that the front-end can be run w/o any server, but if you want to test the auto-redirect, here are some steps?